### PR TITLE
fix runasgroup psp policy

### PR DESCRIPTION
**Pull request by krmayankk**
fixes https://github.com/kubernetes/kubernetes/issues/76105
/sig auth

### CODE REVIEW

File: content/en/docs/concepts/policy/pod-security-policy.md [hunk 1]
```diff
@@ -493,8 +493,8 @@ recommended with this strategy.
 
 - *MustRunAs* - Requires at least one `range` to be specified. Uses the
 minimum value of the first range as the default. Validates against all ranges.
-- *MustRunAsNonRoot* - Requires that the pod be submitted with a non-zero
-`runAsUser` or have the `USER` directive defined (using a numeric GID) in the
+- *MayRunAs* - Does not Require that RunAsGroup be specified. However, when RunAsGroup are
```
Comment (narcher-cs) [inline content/en/docs/concepts/policy/pod-security-policy.md]: ...

**Comment by bradtopol** [comment]
/cc @liggitt 
Hi @liggitt can you do a quick technical review on this. If you feel it is correct just add a /lgtm and I'll take care of the rest.  THANKS

File: content/en/docs/concepts/policy/pod-security-policy.md [hunk 2]
```diff
@@ -493,8 +493,8 @@ recommended with this strategy.
 
 - *MustRunAs* - Requires at least one `range` to be specified. Uses the
 minimum value of the first range as the default. Validates against all ranges.
-- *MustRunAsNonRoot* - Requires that the pod be submitted with a non-zero
-`runAsUser` or have the `USER` directive defined (using a numeric GID) in the
+- *MayRunAs* - Does not Require that RunAsGroup be specified. However, when RunAsGroup are 
+specified, they have to fall in the defined range.
 image. No default provided. Setting `allowPrivilegeEscalation=false` is strongly
```
Comment (bradtopol) [inline content/en/docs/concepts/policy/pod-security-policy.md]: It looks like  "image. No default provided." was left in by accident. Can you remove this

File: content/en/docs/concepts/policy/pod-security-policy.md (hunk 1 above)
Comment (bradtopol) [inline content/en/docs/concepts/policy/pod-security-policy.md]: Instead of are specified it should be is specified

**Comment by bradtopol** [comment]
Hi @krmayankk please see my inline comments. Just a few typos that need to be fixed and then I will merge this.  Thanks

**Comment by krmayankk** [comment]
@bradtopol  fixed

**Comment by Rajakavitha1** [comment]
@bradtopol could you please take a look at the fixes that @krmayankk has made and state your approval.

File: content/en/docs/concepts/policy/pod-security-policy.md [hunk 3]
```diff
@@ -493,10 +493,8 @@ recommended with this strategy.
 
 - *MustRunAs* - Requires at least one `range` to be specified. Uses the
 minimum value of the first range as the default. Validates against all ranges.
-- *MustRunAsNonRoot* - Requires that the pod be submitted with a non-zero
-`runAsUser` or have the `USER` directive defined (using a numeric GID) in the
-image. No default provided. Setting `allowPrivilegeEscalation=false` is strongly
-recommended with this strategy.
+- *MayRunAs* - Does not Require that RunAsGroup be specified. However, when RunAsGroup
```
Comment (sftim) [inline content/en/docs/concepts/policy/pod-security-policy.md]:
```suggestion
- *MayRunAs* - Does not require that RunAsGroup be specified. However, when RunAsGroup
```

I'm not sure whether “Require” in initial-caps was deliberate?

**Comment by tfogo** [comment]
Looks like all suggestions been addressed.

/lgtm
/approve

**Pull request merged**